ref/Microsoft.Build.Utilities.Core/net/Microsoft.Build.Utilities.Core.cs

```diff
@@ -318,6 +318,7 @@ public enum TargetDotNetFrameworkVersion
         Version462 = 10,
         Version47 = 11,
         Version471 = 12,
+        Version472 = 13,
```

Comment:
Shouldn't `VersionLatest` get updated?

Reply:
After reading the comment, maybe bump it to 47 :)

Reply:
The philosophy we've been going with is "leave it where it is for a full major version", per #1728 and https://github.com/Microsoft/msbuild/pull/2637#discussion_r144913003.